### ISSUE TITLE
Forecast Days Data Missing

### ISSUE THREAD

**Issue by macjeff** [bpaauwe/udi-climacell-poly#1]
My weather data is fine and looks great but forecast days are all the same.  I tried 0-15 although not sure if the FREE API includes more than 5 or 7 but they all say Monday and they all have zeros for the data.

<img width="1323" alt="Screen Shot 2020-04-10 at 10 37 41 AM" src="https://user-images.githubusercontent.com/20373758/78998800-5cd2bb00-7b17-11ea-8592-8e35ae98ed93.png">

**Comment by bpaauwe** [comment]
It's working fine for me.  You need to provide debug logs and your configuration.

**Comment by macjeff** [comment]
I think the issue has to do with the 10 min waiting period for forecasts.   I assumed that on restart of the NodeServer it would update data right away but it does not.  (Why cant it?)  I originally had 5 day forecast.  I went out for an hour and that is updated.  So now I added all 15 days that it says it supports.  I restarted the node server and now have to wait again but here is log as of now.

10 min have passed.  I do see something in logs about 12 day forecast limit but both the Climacell docs and your node config page say 15.   

Still only getting Forecast 0-4


Logs attached.


[Climacell_logs_2020-4-10_163143.zip](https://github.com/bpaauwe/udi-climacell-poly/files/4463322/Climacell_logs_2020-4-10_163143.zip)

**Comment by macjeff** [comment]
Changed the number of days to 12.   Since Forecast 0 is day one then day 12 is Forecast11 so deleted Forecast12,13, 14

Then it came in.   So your log shows max days of 12 but your config page shows max days of 15 as does the API Docs.   So its either a bug in getting day 13-15 or the config page has to be changed to max 12.

**Comment by bpaauwe** [comment]
Changed the max number of days to 15.  Thanks for the report.

**Comment by macjeff** [comment]
This still is not working.  Updated to 1.01.  I changed from 12 to 15 days.  The new 3 days came into ISY but never updated.  The old 12 continued.   So I erased the entire node Server and added it again.  Started with 15 day forecast.  None of the 15 updated.    Changed back to 12 days and restarted node server and instantly the 0-11 updated.

Two logs attached.  First one is after the reinstall with 1.01 and 15 day.   Then the second includes the time when I switched to 12 day.
[Climacell_logs_2020-4-10_212219.zip](https://github.com/bpaauwe/udi-climacell-poly/files/4463855/Climacell_logs_2020-4-10_212219.zip)
[Climacell_logs_2020-4-10_211849.zip](https://github.com/bpaauwe/udi-climacell-poly/files/4463856/Climacell_logs_2020-4-10_211849.zip)
